Detectors/GlobalTrackingWorkflow/src/tpcits-match-workflow.cxx

```diff
@@ -28,6 +30,17 @@ void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
   std::swap(workflowOptions, options);
 }
 
+// the matcher process requires the TPC sector completion to trigger and data on
+// all defined routes
+void customize(std::vector<o2::framework::CompletionPolicy>& policies)
+{
+  // the TPC sector completion policy checks when the set of TPC/CLUSTERNATIVE data is complete
+  // in addition we require to have input from all other routes
+  policies.push_back(o2::tpc::TPCSectorCompletionPolicy("itstpc-track-matcher",
+							o2::tpc::TPCSectorCompletionPolicy::Config::RequireAll,
+							InputSpec{"cluster", o2::framework::ConcreteDataTypeMatcher{"TPC", "CLUSTERNATIVE"}})());
```

Comment:
Hi @davidrohr , I have updated the TPCSectorCompletionPolicy class so that it can be initialized with a flag to require data from all input. A generic helper turned out to be complex, there are some limitations and the resulting logic would not be as intended in all cases. So I decided to add the feature to the sector policy class.